.github/workflows/release-package.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Publish Assets
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  upload-assets:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
+          cache: 'npm'
+      - run: yarn install
+      - run: yarn build
```

Comment:
Nuxtjs is used dev server underhood so build command compile this dev server.
To get the static files we need to call `yarn denerate`